vtex/faststore
# feat(gatsby-theme-store): remove home page from theme

## What's the purpose of this pull request?
This PR removes the home page related code from gatsby-theme-store and moves it directly into the store's repo. This way, we don't rely in the gatsby's shadowing mode and are more compatible with other frameworks

## How to test it?
Make sure the home page is exactly the same in both the PR and in production

https://github.com/vtex-sites/marinbrasil.store/pull/544
https://github.com/vtex-sites/btglobal.store/pull/678
https://github.com/vtex-sites/carrefourbrfood.store/pull/882
https://github.com/vtex-sites/storecomponents.store/pull/1011
